.buildscript/libVersions.gradle

```diff
@@ -0,0 +1,4 @@
+ext {
+    libVersionCode = 1
+    libVersionName = "0.1.0"
+}
```

Comment:
These properties appear twice, however they are used only one place. I believe this file is unnecessary.